README.md

```diff
@@ -1,58 +1,148 @@
 # Edit/Update Action
 
-## Objectives
+## Rails Controller Conventions
 
-1. Use form_tag to build an edit form that submits a PUT to update action
-2. Use form_tag and input helpers to prefill form fields based on an AR instance
-3. Find an instance to edit using a dynamic route
-4. Describe the input hack and how to manually trigger another HTTP method
-5. Manually update an instance based on form data
-6. Use update to mass assign data from params to a post to update redirect to show
-7. Explain the problems with form_tag for an edit page
+Now that you know how to implement a create action the next logical step will be to integrate an edit/update action. As you may have noticed, there is a trend in Rails conventions where the logic for rendering a form is separate from the action that the manages the database record alteration. For example:
 
+* The `new` action in the controller simply rendering the `new` form
 
-## Notes
+* The `create` action is what actually handled the process for inserting the form data into the database
 
-let's build an edit and update form that uses form_tag
+In like fashion, the `edit` and `update` actions have a similar convention:
 
-edit/update is more complex then new/create but we're dealing with existing data.
+* The `edit` action will handle rendering the `edit` form
 
-posts#edit needs to find the post we're talking about which means that /posts/:id/edit route.
+* The `update` action will be the method that updates the database record itself
 
-then we render the edit_form.html
 
-we need to point to a different action with a different non support method
+## Rendering the `edit` form
 
-first, what's the URL or named route for the form_tag? post_path(@post). submit the form with that, we get to post show because the form submitted with a GET/POST request instead of a PUT request.
+To start off, let's draw a `get` route for our edit form, since the form will need to know which record is being edited this will need to be a dynamic route that accepts an `:id` as a parameter so it can be accessed by the controller:
 
-the _input hack passing :method to form_tag automatically embeds input hack in your form.
+```ruby
+get '/post/:id/edit', to: 'posts#edit'
+```
 
-now that we have the form wrapper pointing to the write action/url with the method hack, what else is different about our edit form vs our new form?
+We still need to draw one additional route to handle the `update` action, this route will also need to be dynamic and accept an `:id` as a parameter so the `update` action will know what record is being altered:
```

Comment:
also mention that since this is sending data to the server it's not GET, since it's not creating a new record it's not POST, so it's PUT